target/linux/bcm53xx/image/Makefile

```diff
@@ -216,7 +224,8 @@ define Device/dlink
 endef
 
 define Device/dlink-dir-885l
-  DEVICE_TITLE := D-Link DIR-885L
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DIR-885L
   DEVICE_PACKAGES := $(BRCMFMAC_4366B1) $(USB3_PACKAGES)
   $(Device/dlink)
```

Comment:
Only one device using it, but since others might be added I would still move VENDOR to parent definition.